.github/workflows/create-release.yml

```diff
@@ -1,81 +1,137 @@
-name: Create release
+name: Create Release
 
 on:
     workflow_dispatch:
     push:
         branches:
             - main
+            - tmp_hotfix_branch
 
 env:
+    IS_HOTFIX: ${{ github.ref == 'refs/heads/tmp_hotfix_branch' }}
+    IS_PRERELEASE: ${{ github.event_name == 'push' && github.ref != 'refs/heads/tmp_hotfix_branch' }}
+    IS_MANUAL: ${{ contains(github.event.head_commit.message, 'chore(release)') }}
     NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
     NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
 
 jobs:
-    run-release:
+    agents:
+        name: Runner
+        uses: nrwl/ci/.github/workflows/nx-cloud-agents.yml@v0.14
+        secrets:
+            NX_CLOUD_AUTH_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
+            NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_AUTH_TOKEN }}
+        with:
+            node-version: 20.x
+            yarn-version: 3.6.1
+
+    initialize_cloud:
+        runs-on: ubuntu-latest
+        name: Initialize Nx Cloud
+        steps:
+            - uses: actions/checkout@v3
+            - uses: ./.github/actions/nodejs
+            - name: Initialize the Nx Cloud distributed CI run
+              run: npx nx-cloud start-ci-run
```

Comment:
These are not needed here because styles does not really need Distributed Task Execution